guides/source/asset_pipeline.md

```diff
@@ -656,14 +656,24 @@ JS/CSS is excluded, as well as raw JS/CSS files; for example, `.coffee` and
 `.scss` files are **not** automatically included as they compile to JS/CSS.
 
 If you have other manifests or individual stylesheets and JavaScript files to
-include, you can add them to the `precompile` array in `config/initializers/assets.rb`:
-
-```ruby
-Rails.application.config.assets.precompile += %w( admin.js admin.css )
-```
+include, you can add them yourself:
+
+- **Sprockets 3:** Add files to `precompile` array in `config/initializers/assets.rb`:
+  ```ruby
+  Rails.application.config.assets.precompile += %w( admin.js admin.css )
+  ```
+- **Sprockets 4:** Add files or directories inside `./app/assets/config/manifest.js`
+  ```js
+  //= link_tree ../images
+  //= link_directory ../javascripts .js
+  //= link_directory ../stylesheets .css
+  ... (other files/directories)
+  ```
```

Comment:
```suggestion
- **Sprockets 3:** Add files to `precompile` array in `config/initializers/assets.rb`:

  ```ruby
  Rails.application.config.assets.precompile += %w( admin.js admin.css )
  ```

- **Sprockets 4:** Add files or directories inside `./app/assets/config/manifest.js`

  ```js
  //= link_tree ../images
  //= link_directory ../javascripts .js
  //= link_directory ../stylesheets .css
  ... (other files/directories)
  ```
```

Reply:
Just pushed the changes, also noticed that .gitignore does not have .DS_Store included, is there a reason for it?